src/Inviqa/Emarsys/Api/Request/ContactRequest.php

```diff
@@ -23,8 +23,8 @@ class ContactRequest
     private const REGISTRATION_SOURCE = 6092;
     private const REGISTRATION_DETAIL = 6093;
     private const REGISTRATION_DATE = 6097;
-    private const NEWSLETTER = 6094;
-    private const VIP = 6095;
+    private const REISS_NEWSLETTER = 6094;
```

Comment:
I'm not sure about this. Is this Emarsys API client not supposed to be written so that it is not specific to REISS. Should something like this not be passed in via a configuration option or something?

Reply:
yep, but this was the simplest solution given the timescales when this was first developed and Emarsys through in the curve ball of not being able to use the string names.\

However now that we need different values for Live and non live, then your above suggestion will be exactly what I'm doing as part of

https://inviqa.atlassian.net/browse/REISS-1545